Dockerfile

```diff
@@ -40,6 +40,7 @@ FROM node:16-alpine3.15 AS release
 # GET production code from previous containers
 COPY --from=backend_dependencies /telescope/node_modules /telescope/node_modules
 COPY ./src/backend ./src/backend
+COPY package.json .
```

Comment:
I thought we had this one in line 24 already?

Reply:
Because in this stage we use another image, it doesn't have the stuff from the previous stage.

Reply:
Because in this stage we use another image, it doesn't have the stuff from the previous stage.

Reply:
Ooops. Looks like it's failing CI

Reply:
Ah so it only copies the dependencies from the image `backend-dependencies`

Reply:
Fantastic detective work, love it.